src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexAvatar} from '../';
+import {useMeeting} from '../hooks';
+
+import './WebexLocalMedia.scss';
+
+/**
+ * Webex Local Media component displays the user's local video
+ * previous to joining a meeting from a given meetingID.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexLocalMedia({meetingID, personID}) {
+  const {localVideo} = useMeeting(meetingID);
+  const media =
+    localVideo && Object.entries(localVideo).length > 0 ? (
```

Comment:
Is there a reason to check for `localvideo` properties? If there is no local video, wouldn't it be `null`?

Reply:
this is for the initial render when the object returned from `useMeeting(meetingID)` is `undefined` and `localVideo` is a nested property. Therefore we need to check if `localVideo` or the parent object is defined.

Reply:
You're not checking for parent here though? You already "grabbed" localVideo in the previous line, so it's either `null` or `undefined` at this point.
Wouldn't `localVideo ? ...` be enough?

Reply:
no. we are. that's how destructing works. we cant destruct from undefined.